index.html

```diff
@@ -14,10 +14,31 @@
     <link rel="stylesheet" href="about.css">
     <link rel="stylesheet" href="contact.css">
     <link rel="stylesheet" href="desktop.css">
+    <link rel="stylesheet" href="menu.css">
+    <script defer src="script.js"></script>
     <title>Portfolio</title>
 </head>
 <body>
-    <header>
+
+    <div id="menu">
+    </div>
+
+    <div id="menu-items">
+        <div class="close-btn"><button id="close-menu"><img src="images/Cancel.svg" alt="cancel-button"></button></div>

```

Comment:
- [Optional] Great job for removing scrolling 👍, I think it would be better if you moved the close button to the right top and move the menu list a little bit upper as given Figma design to have better UI 👍
![2023-03-14_11-55](https://user-images.githubusercontent.com/52010789/224949577-1509981b-ce15-4c67-a8e4-7454b09c0d38.png)